include/openssl/evp.h

```diff
@@ -234,6 +234,10 @@ OPENSSL_EXPORT EC_KEY *EVP_PKEY_get1_EC_KEY(const EVP_PKEY *pkey);
    (pkey_id == NID_hqc192) || \
    (pkey_id == NID_hqc256) || \
 0 )
+
+OPENSSL_EXPORT int oqs_verify_sig(EVP_PKEY *bssl_oqs_pkey, const uint8_t *sig,
```

Comment:
This is not a good place for the function prototype: The next time `generate.py` is run, this code is deleted (everything between "OQS_TEMPLATE..._START" and "...END" -> Please move this code right before the "// EVP_PKEY_set_type sets..." comment.